src/main/java/com/eatthepath/pushy/console/PushyConsoleApplication.java

```diff
@@ -41,7 +41,7 @@ public class PushyConsoleApplication extends Application {
     private PushyConsoleController pushyConsoleController;
 
     // Based heavily upon https://softwarei18n.org/using-unicode-in-java-resource-bundles-6220776b6099
-    static final ResourceBundle RESOURCE_BUNDLE = ResourceBundle.getBundle("com/eatthepath/pushy/console/pushy-console");
+    static final ResourceBundle RESOURCE_BUNDLE = ResourceBundle.getBundle("com/eatthepath/pushy/console/pushy-console_en");
```

Comment:
Is this change necessary to support live activities? If not, could we save this for a separate pull request?

Reply:
I don't think its necessary :)

Reply:
That's indeed not necessary for the live activities. I needed it to get the console working and accidently added it to the commit. I will take it out.